code/game/objects/items/implants/implant_mindshield.dm

```diff
@@ -43,6 +45,9 @@
 				to_chat(target, "<span class='notice'>You feel a sense of peace and security. You are now protected from brainwashing.</span>")
 		ADD_TRAIT(target, TRAIT_MINDSHIELD, "implant")
 		target.sec_hud_set_implants()
+		if(deconverted)
+			if(prob(1) || SSevents.holidays && SSevents.holidays[APRIL_FOOLS])
+				target.say("I'm out! I quit! Whose kidneys are these?", forced = TRUE)
```

Comment:
forced is a string, used in logging

Reply:
i dont understand what you mean here

Reply:
oh, wait, its what caused it? aight

Reply:
`forced = "You son of a bitch, he's in."`

Reply:
Log example:
SAY clownman3/(John Testerson) "You son of a bitch! I'm in." FORCED by You son of a bitch, he's in.